src/Engine/ProtoAssociative/ProtoAssociative.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>ProtoAssociative</RootNamespace>
     <AssemblyName>ProtoAssociative</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
```

Comment:
Any reason why this one is different than others?

Reply:
Hmmm. Shouldn't be. I'll check.